### DISCUSSION TITLE
Linking my app to my project on directus cloud

### DISCUSSION THREAD

**Discussion by Posoroko** [directus/directus#15022] [Archive]
Hello!
I've been using directus to develop a project in local, and I now want to link my app to a project I created on directus cloud.  So I changed the url in my DirectusConfig.js file from localhost to the project's address. But It doesnt work! I get a CORS error when logging in.

After 2 years of learning web development, I'm now trying to learn about hosting. I realize it's a complete new world of rules and technologies that opens to me.

I've searched into the docs, but could'nt find the proper way to do it. 
Can someone kindly help me out?

Thanks in advance!

eric

**Comment by rijkvanzanten** [comment]
What is the CORS error you're getting?
